feature/settings/import/src/main/kotlin/app/k9mail/feature/settings/import/ui/SettingsImportViewModel.kt

```diff
@@ -244,7 +243,7 @@ internal class SettingsImportViewModel(
                 updateCloseButtonAndImportStatusText()
             }
 
-            GlobalScope.launch(Dispatchers.IO) {
+            viewModelScope.launch(Dispatchers.IO) {
```

Comment:
We probably want to add `withContext(NonCancellable)` here because we don't want the (background) operation to be canceled if the user navigates away from the screen.